package.json

```diff
@@ -31,6 +31,7 @@
     "react-final-form-arrays": "^3.1.1",
     "react-scripts": "3.4.1",
     "resize-observer-polyfill": "^1.5.1",
+    "slash-sandwich": "^1.0.7",
```

Comment:
Seems I'm not able to find this module on npm? https://www.npmjs.com/search?q=slash-sandwich

Reply:
Made it myself.
Just extracted the slash-sandwich code from `/scripts/` and made an external package out of it.
I also wanted to be able to use in `./src`

(Also added tests)

https://www.npmjs.com/package/slash-sandwich
https://github.com/Jeffrey-Zutt/slash-sandwich